tests/ui/swap.rs

```diff
@@ -5,9 +5,11 @@
     clippy::disallowed_names,
     clippy::no_effect,
     clippy::redundant_clone,
+    clippy::almost_swapped,
```

Comment:
We're testing for this, so we don't want to `#![allow]` it

Reply:
If there are some suggestions that don't compile, create a `swap_unfixable.rs` and put them in there instead